src/core/Table/columns/actionColumn.tsx

```diff
@@ -0,0 +1,79 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { HeaderProps } from 'react-table';
+import { Checkbox } from '../../Checkbox';
+import SvgColumnManager from '@itwin/itwinui-icons-react/cjs/icons/ColumnManager';
+import { DropdownMenu } from '../../DropdownMenu';
+import { IconButton } from '../../Buttons/IconButton';
+import { MenuItem } from '../../Menu';
+import { TABLE_RESIZE_START_ACTION } from '../actionHandlers';
+
+const ACTION_CELL_ID = 'iui-table-action';
+
+export const ActionColumn = <T extends Record<string, unknown>>(
+  hasColumnManager: boolean,
+) => {
+  return {
+    id: ACTION_CELL_ID,
+    disableResizing: true,
+    disableGroupBy: true,
+    minWidth: 48,
+    width: 48,
+    maxWidth: 48,
+    columnClassName: 'iui-slot',
+    cellClassName: 'iui-slot',
+    disableReordering: true,
+    Header: ({ allColumns, dispatch }: HeaderProps<T>) => {
+      const [isOpen, setIsOpen] = React.useState(false);
+      if (!hasColumnManager) {
+        return null;
+      } else {
+        const headerCheckBoxes = () =>
+          allColumns
+            //Filters out any default columns made such as selection and expansion
+            .filter(({ id }) => !id.includes('iui-table'))
+            .map((column) => {
+              const { onChange, checked } = column.getToggleHiddenProps();
+              return (
+                <MenuItem
+                  key={column.id}
+                  icon={
+                    <Checkbox
```

Comment:
Can you add an `aria-label` to this checkbox with the name of the column? Or use `aria-labelledby` to reference the same text that is inside the MenuItem

Reply:
`aria-label` added

Reply:
Hopefully, its always a string...

Reply:
Oh, looks like it won't always be a string.
<img width="484" alt="image" src="https://user-images.githubusercontent.com/9084735/161760277-3c7de7df-0faa-4741-b9e7-b2e6ac4d6cc9.png">

So then we should use `aria-labelledby` on the rendered JSX

Reply:
`aria-labelledby` added. Also, The menu item's inner HTML is rendered using `column.render('Header')` instead of `column.Header` to work with Headers that are strings or return JSX. See [here](https://github.com/iTwin/iTwinUI-react/pull/564/files#diff-e9eae8b7f2f64123d6cf3fcdc63983c5293641819768d19dfaa47ec82399fe88R69)